js/web/docs/webgpu-operators.md

```diff
@@ -50,6 +50,7 @@ Do not modify directly.*
 | Gather | ai.onnx(1-10,11-12,13+) |  |
 | GatherBlockQuantized | com.microsoft(1+) |  |
 | GatherElements | ai.onnx(11-12,13+) |  |
+| GatherND | ai.onnx(11+,12+,13+) |  |
```

Comment:
> ai.onnx(11+,12+,13+) 

This version number range is strange. So after checking the source code, I found that the problem is in the kernel registration.

in `js_execution_provider.cc`, you should use `ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME` for version 11 and version 12 instead of `ONNX_OPERATOR_KERNEL_CLASS_NAME`. Also apply to the lines of `BuildKernelCreateInfo`.